src/main/java/com/salesforce/dynamodbv2/mt/admin/AmazonDynamoDBAdminUtils.java

```diff
@@ -112,6 +111,18 @@ private String getTableStatus(String tableName) throws TableInUseException {
         }
     }
 
+    private boolean tableActive(String tableName) throws TableInUseException {
```

Comment:
what happened here? method (and inner class) reordering?

Reply:
I have no idea.  ctrl-alt-l has a mind of its own.